sample-btcd.conf

```diff
@@ -293,8 +293,7 @@
 ; worth your while.
 ; generate=false
 
-; Add addresses to pay mined blocks to for CPU mining and the block templates
-; generated for the getwork RPC as desired.  One address per line.
+; Add addresses to pay mined blocks to for CPU mining.  One address per line.
```

Comment:
The bit about block templates is still true it's just for `getblocktemplate` instead of `getwork`.